library/spdm_responder_lib/libspdm_rsp_communication.c

```diff
@@ -23,6 +24,278 @@ libspdm_return_t libspdm_responder_dispatch_message(void *spdm_context)
 
     context = spdm_context;
 
+#if LIBSPDM_FIPS_MODE
+
+#if LIBSPDM_FIPS_TEST_AT_LOAD
```

Comment:
Is the following selftest code same as in libspdm_init_connection()?  Why duplicate the code?

Reply:
I have added a api `bool libspdm_update_fips_selftest_context(void *spdm_context)`, which can be called by requester and responder. Thanks.